adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -134,7 +133,7 @@ void HybrisProximityAdaptor::processSample(const sensors_event_t& data)
     d->withinProximity_ = near;
     d->value_ = data.distance;
 
-    lastNearValue = near;
+    lastNearValue = (int)near;
```

Comment:
I would assume bool gets promoted to int without problems. If the domain difference needs to be highlighted for some reason, a comment might be nicer than an unnecessary cast?